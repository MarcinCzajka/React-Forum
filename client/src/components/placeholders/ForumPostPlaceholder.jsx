import React from 'react';
import { Placeholder } from 'semantic-ui-react';
import './ForumPostPlaceholder.scss';

export default () => {
    return (
        <div className='postPlaceholder' >
            <Placeholder className='imageOverlay' >
                <Placeholder.Image />
            </Placeholder>
            <Placeholder >
                <Placeholder.Line />
                <Placeholder.Line />
                <Placeholder.Line />
                <Placeholder.Line />
            </Placeholder>
        </div>
    )
}