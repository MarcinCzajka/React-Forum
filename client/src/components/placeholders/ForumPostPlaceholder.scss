.postPlaceholder {
    display: flex;

    :first-child {
        height: 150px;
        width: 220px;
    }  
    
    :last-child {
        width: 100%;
        margin: 10px;
    }

}