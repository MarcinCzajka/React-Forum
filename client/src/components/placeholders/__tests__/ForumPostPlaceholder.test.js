import React from 'react';
import { create } from "react-test-renderer";
import ForumPostPlaceholder from '../ForumPostPlaceholder';

describe('Captcha component', () => {
    test('match snapshot', () => {
        const component = create(<ForumPostPlaceholder />);
        expect(component).toMatchSnapshot();
    });

});