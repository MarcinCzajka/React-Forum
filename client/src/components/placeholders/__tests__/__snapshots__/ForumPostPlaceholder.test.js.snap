// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Captcha component match snapshot 1`] = `
<div
  className="postPlaceholder"
>
  <div
    className="ui placeholder"
  >
    <div
      className="image"
    />
  </div>
  <div
    className="ui placeholder"
  >
    <div
      className="line"
    />
    <div
      className="line"
    />
    <div
      className="line"
    />
    <div
      className="line"
    />
  </div>
</div>
`;
